binary/expression.proto

```diff
@@ -148,32 +148,31 @@ message Expression {
     }
   }
 
-  message IfThen {
-
-    repeated IfClause ifs = 1;
+  message CasePredicateExpression {
```

Comment:
Can we keep the IfThen names as opposed to changing this? I think it does a better job of clarifying distinction from the other type. It also avoids being overly sql centric for a pretty standard programming concept.

Reply:
That's fair. I can keep the original names.

Reply:
I think that CaseValue (or SwitchValue) are an improvement over simply "switch". Just felt that ifthen was probably clearer to most people.

Reply:
Done